instrumentation/bunny/lib/opentelemetry/instrumentation/bunny/patches/channel.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Bunny
+      module Patches
+        # The Channel module contains the instrumentation patch the Channel#basic_get, Channel#basic_publish and Channel#handle_frameset methods
+        module Channel
+          def basic_get(queue, opts = { manual_ack: false })
+            delivery_info, properties, payload = super
```

Comment:
If `super` [raises](https://github.com/ruby-amqp/bunny/blob/master/lib/bunny/channel.rb#L596) here do we not want this event captured in the trace?